### PR TITLE
Add Subnet link in the issue template for urgency clause

### PR DESCRIPTION
**Pull request by vadmeste**
An update of the bug report template to show how to get quicker response for urgent issues.

### CODE REVIEW

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

File: .github/ISSUE_TEMPLATE.md [hunk 1]
```diff
@@ -44,3 +44,6 @@ assignees: ''
 * Version used (`minio --version`):
 * Server setup and configuration:
 * Operating System and version (`uname -a`):
+
+## Urgency
+If this case is urgent, please subscribe to [Subnet](https://min.io/pricing) so that our 24/7 support team may help you faster.
```
Comment (harshavardhana) [inline .github/ISSUE_TEMPLATE.md]: Let's move this to the top of the template @vadmeste as a NOTE:

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -44,3 +44,6 @@ assignees: ''
 * Version used (`minio --version`):
 * Server setup and configuration:
 * Operating System and version (`uname -a`):
+
```
Comment (harshavardhana) [inline .github/ISSUE_TEMPLATE/bug_report.md]: Let's move this to the top of the template @vadmeste as a NOTE:

**Review by harshavardhana — CHANGES_REQUESTED** [review]

**Comment by minio-trusted** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-gateway-s3.sh | :heavy_check_mark: |
| mint-erasure.sh | :heavy_check_mark: |
| mint-dist-erasure.sh | :heavy_check_mark: |
| mint-gateway-nas.sh | :heavy_check_mark: |
| mint-compress-encrypt-dist-erasure.sh | :heavy_check_mark: |
| mint-pools.sh | :heavy_check_mark: |
Deleting image on docker hub
Deleting image locally

**Review by harshavardhana — APPROVED** [review]

**Pull request merged**